Cognigy/WebchatWidget
# feature/17638 remove useTitleAsPayload feature

Description:

This PR removes the feature to use the title field as payload for quick replies. 

This will revert this merged PR: https://github.com/Cognigy/WebchatWidget/pull/211

Success criteria:

The payload should still be used instead of the title

How to test (steps):

1. Create a say node with quickreplies that have a title different than it's payload
2. Create a webchat endpoint and add this custom json:
```json
{
    "useQuickReplyTitleAsPostback":true
}
```
3. Chat with your endpoint and click the quickreply, the response should still be the payload and not the title

Additional considerations (describe):

  - Examples: adding, renaming, removing a field or changing the format or constraints of a field

Security implications (describe):
